src/components/view/sources/sources.css

```diff
@@ -18,17 +25,28 @@
     color: #70d6ff;
     transition: 0.25s;
     cursor: pointer;
+    position: relative;
+    z-index: 3;
+    transition: 0.5s linear;
 }
 
 .source__item:hover,
 .source__item:focus {
     border-color: #3fcc59;
     color: #69db7e;
     box-shadow: 0 0.5em 0.5em -0.4em #3fcc59;
-    transform: translateY(-0.25em);
+    transform: scale(1.2);
 }
 
 .source__item-name {
     font-weight: 400;
     white-space: nowrap;
 }
+
+@media (max-width: 799px) and (min-width: 300px) {
+
+    .sources {
+        justify-content: center;
+        
+    }
```

Comment:
Please be consistent in formatting and using whitespace

Reply:
Removed whitespaces.